src/sql/base/browser/ui/panel/panelStyles.ts

```diff
@@ -49,20 +53,17 @@ registerThemingParticipant((theme: ITheme, collector: ICssStyleCollector) => {
 	// Styling with Outline color (e.g. high contrast theme)
 	const outline = theme.getColor(activeContrastBorder);
 	if (outline) {
-		const outline = theme.getColor(activeContrastBorder);
-
 		collector.addRule(`
-			.tabbedPanel > .title > .tabList .tab .tabLabel.active,
-			.tabbedPanel > .title > .tabList .tab .tabLabel:hover {
+			.tabbedPanel > .title > .tabList .tab-header.active,
```

Comment:
these panel styles also affect the other panels, is this negatively affecting panels in restore?

Reply:
No it wouldn't. I tested in both restore dialog and result grid

---
In reply to: [167041313](https://github.com/Microsoft/sqlopsstudio/pull/629#discussion_r167041313) [](ancestors = 167041313)

Reply:
oops I did regress panels in restore

---
In reply to: [167045996](https://github.com/Microsoft/sqlopsstudio/pull/629#discussion_r167045996) [](ancestors = 167045996,167041313)